docker-compose-services/minio/docker-compose.minio.yaml

```
@@ -0,0 +1,36 @@
+version: '3.6'
+
+services:
+  minio:
+    container_name: ddev-${DDEV_SITENAME}-minio
+    build:
+      context: './minio-build'
+    restart: always
+    ports:
```

Please use `expose:` here instead of `ports:`.  That will make it not create a bind point on the host, which is unneeded. 
```suggestion
    expose:
```